Shared/Presenter/BaseItemListPresenter.swift

```diff
@@ -21,6 +21,7 @@ struct LoginListTextSort {
     let sortingOption: Setting.ItemListSort
     let syncState: SyncState
     let storeState: LoginStoreState
+    let networkConnected: Bool
```

Comment:
At some point this `struct` should change name.

`LoginListTextSort` doesn't really capture why it should have `networkConnected` in it.